errors/src/storage/storage.rs

```diff
@@ -20,12 +20,8 @@ use crate::{
     parameters::ParametersError,
 };
 
-use bincode;
 use std::fmt::Debug;
 
-#[cfg(feature = "librocksdb")]
```

Comment:
I believe this is needed for our WASM compilations to work.

Reply:
`rocksdb` is the name of the dependency, so it is automatically available with this feature enabled - it's the same principle as with `bincode` not requiring an explicit `use` import